code/game/turfs/simulated/walls.dm

```diff
@@ -1,6 +1,6 @@
 /turf/closed/wall
 	name = "wall"
-	desc = "A huge chunk of metal used to separate rooms."
+	desc = "A huge chunk of metal used to separate rooms.\n<span class='notice'>The outer plating is <b>welded</b> firmly in place.</span>"
```

Comment:
Not sure if this works with any other exotic walls, but works ok for the normal station interior?

Reply:
yeah making the decon hints a proc called in base wall examine with an rwall override would be better by far

Reply:
I made separate procs for wall and rwall:
/turf/closed/wall/proc/deconstruction_hints(mob/user)
/turf/closed/wall/r_wall/proc/deconstruction_hints_rwall(mob/user)

And in rwall-examine:
/turf/closed/wall/r_wall/examine(mob/user)
	. = ..()
	deconstruction_hints_rwall(user)

But they still get both called on rwalls? How do I override the wall's examine?